src/script/mod.rs

```diff
@@ -413,6 +415,86 @@ impl Output {
 
         Ok(())
     }
+
+    /// Create build scripts without running them
+    pub async fn create_build_script(&self) -> Result<(), std::io::Error> {
```

Comment:
Is it possible to remove code duplication for this feature and create the script with the original code but just skip execution, perhaps?